iree/compiler/Dialect/HAL/Target/LLVM/LLVMAOTTarget.cpp

```diff
@@ -146,8 +149,13 @@ class LLVMAOTTargetBackend final : public TargetBackend {
   void getDependentDialects(DialectRegistry &registry) const override {
     mlir::registerLLVMDialectTranslation(registry);
     // TODO: make inclusion of ArmNeon conditional?
-    registry
-        .insert<IREE::Codegen::IREECodegenDialect, arm_neon::ArmNeonDialect>();
+    // clang-format off
+    registry.insert<IREE::Codegen::IREECodegenDialect,
```

Comment:
I am surprised you need this, cause adding these to `LLVMCPULowerExecutableTarget` should have been enough for using with `iree-opt`.

I see you already updated https://github.com/google/iree/blob/main/iree/tools/init_mlir_dialects.h. THat should have been enough. Do you know what fails if you dont add these here.

Reply:
The first test I had was using iree-translate / iree-compile.
I can remove here for the sake of being more self-contained but things will not work end to end for now.
I'll reactivate when I reconnect e2e execution.